test/runtests.jl

```diff
@@ -63,11 +63,6 @@ facts("Tests: dot{x}=x^2, x(0) = 3; nsteps <= maxsteps") do
     @fact length(tv) --> 14
     @fact length(xv[:,1]) --> 14
     @fact length(xv[:,2]) --> 14
-    if VERSION < v"0.5-"
-        @fact xv[1,1:end] --> q0'
-    else
-        @fact xv[1,1:end] --> q0
-    end
```

Comment:
The line `@fact xv[1,1:end] --> q0` should remain there, i.e., where the Julia version is at least 0.5? The same on the other changes below in the file.